test/specs/instanceApi.unit.spec.js

```diff
@@ -199,41 +199,49 @@ describe('instance api', function () {
       expect(nodeCallback.calls.count()).toBe(1)
     })
 
-    xit('doesn\'t call node callback after callback is removed', function () {
-      var nodeCallback = jasmine.createSpy('node callback')
-      var ascent = list(namedNode('node', {}))
+    describe('after callback is removed', function () {
+      beforeEach(function () {
+        oboeBus.un.and.callThrough()
```

Comment:
Could you talk me through how this fix works? I think what's happening is that we've mocked these functions so that we can spy on them, but we need them to be called through to the actual implementation so that things will actually work

Reply:
Pretty much. They are mocked above at the very top of the tests in the first `beforeEach`. There is no implementation given so they are effectively no-ops. 

However the test itself expects the `un` function to work as expected when `removeListener` is called. So we have to let it `callThrough` to actually remove the callback.